apps/mobile/src/components/ChatDetailScreen/components/VexlbotMessageItem/components/TradeChecklistAmountView.tsx

```diff
@@ -0,0 +1,243 @@
+                    ? t('vexlbot.higherThanLivePrice')
+                    : t('vexlbot.lowerThanLivePrice')
+                }`
+              : undefined
+          }
+          text={t(
+            amountDataToDisplay.status === 'pending'
+              ? 'vexlbot.settledAmountOfTheDeal'
+              : 'vexlbot.suggestedAmountOfTheDeal',
+            {
+              username:
+                amountDataToDisplay.by === 'me'
+                  ? t('common.you')
+                  : otherSideData.userName,
+              btcAmount: amountDataToDisplay.amountData.btcAmount,
+              fiatAmount: amountDataToDisplay.amountData.fiatAmount,
+              fiatCurrency:
+                currencies[offerForChat?.offerInfo.publicPart.currency ?? 'USD']
+                  .code,
+              feeAmount: amountDataToDisplay.amountData.feeAmount,
+            }
+          )}
+        >
+          <XStack f={1} ai={'center'} jc={'space-between'}>
+            {amountDataToDisplay.by === 'them' &&
+            amountDataToDisplay.status === 'pending' ? (
+              <Button
+                fullWidth
+                disabled={!amountData?.received}
+                onPress={onAcceptButtonPress}
+                variant={'secondary'}
+                size={'small'}
+                text={t('common.accept')}
+              />
+            ) : (
+              <>
+                {amountDataToDisplay.amountData.btcAmount &&
+                  amountDataToDisplay.status !== 'pending' && (
+                    <Button
+                      text={'BTC'}
+                      beforeIcon={copySvg}
+                      onPress={() => {
+                        Clipboard.setString(
+                          `${amountDataToDisplay.amountData.btcAmount}`
+                        )
+                      }}
+                      size={'small'}
+                      variant={'primary'}
+                      iconFill={getTokens().color.main.val}
+                    />
+                  )}
+                {amountDataToDisplay.amountData.btcAmount &&
+                  amountDataToDisplay.status !== 'pending' && (
+                    <Button
+                      text={'SAT'}
+                      beforeIcon={copySvg}
+                      onPress={() => {
+                        Clipboard.setString(
+                          `${
+                            amountDataToDisplay.amountData.btcAmount
+                              ? Math.round(
+                                  amountDataToDisplay.amountData.btcAmount *
+                                    SATOSHIS_IN_BTC
+                                )
+                              : 0
+                          }`
+                        )
+                      }}
+                      size={'small'}
+                      variant={'primary'}
+                      iconFill={getTokens().color.main.val}
+                    />
+                  )}
```

Comment:
create (()=>{...})() function above and render the result here to make the conditions more readable